charts/primary-site/values.yaml

```diff
@@ -18,6 +18,13 @@ globals:
   inbox:
     storageProvider: google_cloud
     bucketName: foxglove-inbox
+  
+  proxy:
+  ## If you are running behind a proxy, otherwise set this to 'false'
+    enabled: true
```

Comment:
@jeff-foxglove it seems weird to me that we defaulted this to _true_ and tell people to set it to false. The comment and decision seems backwards. Proxy should be the exception not the default.